### ISSUE TITLE
More useful commit message and pull request title?

### ISSUE THREAD

**Issue by jetersen** [olblak/updatecli#53]
**Is your feature request related to a problem? Please describe.**

First off, I would be happy to implement it, though just wanted some input.

Currently the commit message looks like this:
```
[updatecli] Key 'versions.certManager', from file 'system-services/helm/values.yaml', was updated to 'v0.15.1'
```

and pull-request title looks like this
```
[Updatecli] Update version to v0.15.1
```

At the moment I find the commit message rather verbose yet not descriptive enough.

Perhaps target should require `name` or perhaps source can be used to derive the name?
I think a commit message using a description as well would be more helpful
```
[updatecli] Update cert-manager version to v0.15.1

Key 'versions.certManager', from file 'system-services/helm/values.yaml', was updated to 'v0.15.1'
```

pr title is currently also very vague with multiple PRs open it not clearly exactly what is being updated.
```
[Updatecli] Update cert-manager version to v0.15.1
```

With the config below both `source.spec.name` and `targets.id.name` can be used

```tpl
source:
  kind: helmChart
  spec:
    url: https://charts.jetstack.io
    name: cert-manager
targets:
  certManager:
    name: cert-manager
    kind: yaml
    spec:
      file: system-services/helm/values.yaml
      key: versions.certManager
    scm:
      github:
        user: "{{ .github.user }}"
        email: "{{ .github.email }}"
        owner: "org"
        repository: "repo"
        token: "{{ requiredEnv .github.token }}"
        username: "{{ .github.username }}"
        branch: "master"
```

![image](https://user-images.githubusercontent.com/1661688/86027971-9b4e8680-ba31-11ea-8a7a-f76e22a95002.png)

**Comment by jetersen** [comment]
Perhaps branch name could also benefit from using source/target name.

**Comment by olblak** [comment]
I totally agree with you, the PR body is defined [here](https://github.com/olblak/updatecli/blob/master/pkg/github/main.go#L301L355) as you can see it 's really basic. I don't have strong opinions on how to solve it.

**Comment by olblak** [comment]
Solved by #54
